templates/maas/index.html

```diff
@@ -0,0 +1,1060 @@
+{% extends "base_index.html" %}
+
+{% block title %}Metal as a Service{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1Gv3_A3vW2k7U-0B0KVSHkEnGwgtIlDl5P1jGMSc8sqI/edit
+{% endblock meta_copydoc %}
+
+{% block content %}
+  <section class="p-takeover--dark">
+    <div class="row">
+      <div class="col-7">
+        <h1>Very fast server provisioning for your data centre</h1>
+        <p>
+          Self-service, remote installation of Windows, CentOS, ESXi and Ubuntu on real servers turns your data centre into a bare metal cloud.
+        </p>
+        <p>Welcome to metal-as-a-service.</p>
+        <ul class="p-inline-list u-no-margin--bottom">
+          <li class="p-inline-list__item">
+            <a href="/maas/tutorials/build-a-maas-and-lxd-environment-in-30-minutes-with-multipass-on-ubuntu#1-overview"
+               class="p-button--positive"
+               itemprop="url">Try  MAAS on your PC in minutes</a>
```

Comment:
```suggestion
               itemprop="url">Try MAAS on your PC in minutes</a>
```
This tutorial link is throwing 404